File: CPP/beginner/1005.cpp
#include <stdio.h>
 
int main() {
 
    double A, B, MEDIA = 0;
    double pesoA = 3.5, pesoB = 7.5;
    
    scanf("%lf", &A);
    scanf("%lf", &B);
    
    MEDIA = ((A * pesoA) + (B * pesoB)) / (pesoA + pesoB);
    
    printf("MEDIA = %.5lf\n", MEDIA);
    
    return 0;
}

File: CSharp/beginner/1005.cs
using System;

class Program
{
    static void Main(string[] args)
    {

        double pesoA = 3.5;
        double pesoB = 7.5;

        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());

        double media = ((a * pesoA) + (b * pesoB)) / (pesoA + pesoB);

        Console.WriteLine($"MEDIA = {media:F5}");
    }
}
